facia-press/app/frontpress/PressedCollectionVisibility.scala

```diff
@@ -0,0 +1,60 @@
+package frontpress
+
+import layout.slices.Container
+import model.PressedCollectionVersions
+import model.facia.PressedCollection
+import model.pressed.PressedContent
+
+
+case class PressedCollectionVisibility(pressedCollection: PressedCollection, visible: Int) extends implicits.Collections {
+  lazy val affectsDuplicates: Boolean = Container.affectsDuplicates(pressedCollection.collectionType, pressedCollection.curatedPlusBackfillDeduplicated)
+  lazy val affectedByDuplicates: Boolean = Container.affectedByDuplicates(pressedCollection.collectionType, pressedCollection.curatedPlusBackfillDeduplicated)
+
+  lazy val deduplicateAgainst: Seq[String] = {
+    if (affectsDuplicates)
+      pressedCollection
+        .curatedPlusBackfillDeduplicated
+        .take(visible)
+        .filter(_.participatesInDeduplication)
+        .map(_.header.url)
```

Comment:
best to use `_.properties.webUrl` I think

Reply:
It used to be de-duplicated against `header.url`

Reply:
nevermind then.